src/components/SingleName/NameRegister/Explainer.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import moment from 'moment'
+
+import mq from 'mediaQuery'
+import Step from './Step'
+import Button from '../../Forms/Button'
+import { ReactComponent as Bell } from '../../Icons/Bell.svg'
+import { ReactComponent as Tick } from '../../Icons/GreyCircleTick.svg'
+
+import { requestPermission, hasPermission } from './notification'
+
+const Steps = styled('section')`
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 30px;
+  ${mq.large`
+    grid-template-columns: repeat(3, 1fr);
+    grid-column-gap: 30px;
+    grid-template-rows: 1fr;
+  `}
+`
+
+const Header = styled('div')`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+
+  h2 {
+    font-family: Overpass;
+    font-weight: 300;
+    font-size: 18px;
+    color: #2b2b2b;
+    letter-spacing: 0;
+    margin: 0;
+    margin-bottom: 5px;
+
+    ${mq.medium`
+      font-size: 24px;
+      color: #2B2B2B;
+      letter-spacing: 0;
+    `}
+  }
+
+  p {
+    margin: 0;
+    font-weight: 400;
+    font-family: Overpass;
+    font-size: 14px;
+    color: #adbbcd;
+    letter-spacing: 0;
+  }
+`
+
+const NotifyButton = styled(Button)`
+  flex-shrink: 0;
+`
+
+const NotifyButtonDisabled = styled('div')`
+  color: hsla(0, 0%, 82%, 1);
+`
+
+const Explainer = ({ step, waitPercentComplete, waitTime }) => {
+  const titles = {
+    PRICE_DECISION: 'Registering a name requires you to complete 3 steps',
+    COMMIT_SENT:
+      'Don’t close your browser! You’ll be able to register your name soon.',
```

Comment:
Maybe you want to put this into constant rather than repeating 4 times?

Reply:
Don't think it's necessary to create another constant. This is already a constant technically and an abstraction makes it harder to read